### DISCUSSION TITLE
About ABP detection data

### DISCUSSION THREAD

**Discussion by gen3111620** [nv-morpheus/Morpheus#1884] [Morpheus]
Hi everyone, may I know what is used to collect the data of the packet? If I want to collect it as well, do I need to use other Nvidia products in conjunction?

**Comment by efajardo-nv** [comment]
Hi @gen3111620 We used [Wireshark](https://www.wireshark.org/) to collect the packet data. No, it does not require other Nvidia products to be used in conjunction.
